test/requirements.txt

```diff
@@ -0,0 +1,439 @@
+#
```

Comment:
Why do you have so many packages here? It should just be the few additional packages from the circleci config?

Reply:
We are using [pip-tools](https://pypi.org/project/pip-tools/) and it does save all the dependencies with their versions under requirements.txt, it is much better to have all dependency versions saved for a library.